content/docs/Other guides/monitoring-with-prometheus-loki-and-grafana.md

```diff
@@ -0,0 +1,115 @@
+---
+title: "Monitoring with Prometheus, Loki, and Grafana"
+linkTitle: "Monitoring with Prometheus, Loki, and Grafana"
+weight: 5
+date: 2021-08-01
+description: >
+  Configuring OpenCue with external monitoring services
+---
+
+This page describes how to stand up a sample OpenCue deployment configured to send data to external
+monitoring services.
+
+The sandbox deployment described here utilizes:
+
+*   Prometheus for collecting metrics from the OpenCue database and scheduler.
+*   Loki for collecting logs from all OpenCue components.
+*   Grafana to provide an interface for querying and displaying the data send by the others.
+
+## Before you begin
+
+*  Follow our quick start guide, which will walk you through standing up a basic OpenCue
+   deployment using Docker compose. This guide builds on the quick start and they share the same
+   prerequisites.
+*  Install the [Loki Docker driver](https://grafana.com/docs/loki/latest/clients/docker-driver/).
+
+## Deploying the OpenCue with monitoring sandbox environment
```

Comment:
'the OpenCue' reads a little awkwardly to me. I think the description version is clearer, e.g.:

Deploying OpenCue with external monitoring services

Reply:
Agree, struggled to find the best wording here. I like yours, done.